### DISCUSSION TITLE
🤔🤔 Apply Appropriate font family

### DISCUSSION THREAD

**Discussion by price76** [befrooshim/com.befrooshim.window#16] [Ideas]
For better design and turn number to right format is good to apply a suitable font family. It can effect on UX too. So i think **IRANSans** is the best choice to this job for this project.

**Comment by mohsenxad** [comment]
Add **IranSans** font to **src/assets/fonts** folder

**Comment by mohsenxad** [comment]
@price76 **IranSans** Font is available in the **src/assets/folder** in **main** branch by #19 and #18
